addons/vrm/import_vrm.gd

```diff
@@ -864,11 +864,11 @@ func _import_scene(path: String, flags: int, bake_fps: int):
 	return packed_scene.instance(PackedScene.GEN_EDIT_STATE_INSTANCE)
 
 
-func import_animation_from_other_importer(path: String, flags: int, bake_fps: int):
+func _import_animation_from_other_importer(path: String, flags: int, bake_fps: int):
 	return self._import_animation(path, flags, bake_fps)
 
 
-func import_scene_from_other_importer(path: String, flags: int, bake_fps: int):
+func _import_scene_from_other_importer(path: String, flags: int, bake_fps: int):
 	return self._import_scene(path, flags, bake_fps)
```

Comment:
Please remove this code instead

It is unused and there is no use in adding a `_` to an unused function.